bgc_data/bgc_zooplankton_abundance_copepods_data.sql

```diff
@@ -0,0 +1,60 @@
+-- Materialized view for Zooplankton Copepods product
+-- To be served as a WFS layer by Geoserver using output format csv-with-metadata-header,
+-- which will convert the josnb `abundances` column into separate CSV columns on output.
+CREATE MATERIALIZED VIEW bgc_zooplankton_abundance_copepods_data AS
+WITH grouped AS (
+    -- join changelog on to raw data, pick only rows where copepod identified + other cases identified in Claire code
+    -- group by trip, genus, species and changelog details
+	-- select only copepods
+    SELECT r.trip_code,
+	   CONCAT(r.genus, ' ', r.species) AS composite_species_name, 
+           c.startdate,
+           substring(taxon_name, '^\w+') != substring(parent_name, '^\w+') AS name_changed,
```

Comment:
@clairedavies For this species-level product, shouldn't this just be a full comparison? i.e.
```suggestion
           taxon_name != parent_name AS name_changed,
```